jupyter-config/cern-config/cern_bundles.json

```diff
@@ -0,0 +1,288 @@
+{
```

Comment:
Exactly, feel free to clean in this PR or later

Reply:
Yes, just a simple example would be enough, I'd remove all CERN-related configuration details of the bundles. This will in the end this will all be [here](https://gitlab.cern.ch/db/hadoop-confext/-/blob/master/bundles/bundles.json), and the extension should be configurable to pick it from there.

Reply:
Okay I'll clean the unused files